lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the concept of ulimit increase allows more file descriptors to be opened for some of the unit tests in REEF
```

Comment:
The wiki is only for running java builds, not for compiling, should we still have a wiki separately for running java builds?